src/providers/twitch/api/Helix.hpp

```diff
@@ -329,6 +330,16 @@ struct HelixChatSettings {
     }
 };
 
+class HelixUserList {
+    public:
+        std::unordered_set<QString> users;
+        int total;
```

Comment:
Preferably `total` isn't included in this data structure to keep it as small/to-the-point as possible

Reply:
Ok, I don't think I need this class then, I'll just pass back the unordered set.

Reply:
Yeah that works well :+1: